tests/OmniSharp.Roslyn.CSharp.Tests/DiagnosticsFacts.cs

```diff
@@ -25,21 +25,34 @@ public async Task CodeCheckSpecifiedFileOnly()
             var requestHandler = GetRequestHandler(SharedOmniSharpTestHost);
             var quickFixes = await requestHandler.Handle(new CodeCheckRequest() { FileName = "a.cs" });
 
-            Assert.Single(quickFixes.QuickFixes);
+            Assert.Contains(quickFixes.QuickFixes.Select(x => x.ToString()), x => x.Contains("CS0029"));
```

Comment:
Since this went from one to many diagnostics, what are the new ones that show up? Should we be explicit about what they are?

Reply:
Specific test case tests CodeCheckSpecifiedFileOnly which purpose is fulfilled when checked that errors from specific code is not returned. I don't feel specifying other errors give much value at that case. However adding them makes test unnecessarily fragile since cases like updating to new roslyn -> new analyzers are added -> broken tests even when everything works as expected. In few test this isn't that much problem but i fear that when more and more tests are very specific they just keep breaking up when roslyn is updated.